### ISSUE TITLE
TWO ISSUES: Matching Triton not found - How to activate XFORMERS?

### ISSUE THREAD

**Issue by LIQUIDMIND111** [pinokiocomputer/pinokio#62]
Hello. When I start my app I get this error:

A matching Triton is not available, some optimizations will not be enabled.
Error caught was: No module named 'triton'
C:\Users\Jengibre\pinokio\api\svd.pinokio.git\generative-models\venv\lib\site-packages\streamlit\watcher\local_sources_watcher.py:177: UserWarning: T
orchaudio's I/O functions now support par-call bakcend dispatch. Importing backend implementation directly is no longer guaranteed to work. Please us
e `backend` keyword with load/save/info function, instead of calling the udnerlying implementation directly.
  lambda m: [p for p in m.__path__._path],
file_path = C:\Users\Jengibre\pinokio\api\svd.pinokio.git\generative-models\checkpoints\svd.safetensors
File already exists at C:\Users\Jengibre\pinokio\api\svd.pinokio.git\generative-models\checkpoints\svd.safetensors

ALSO, I remember the app installed XFORMERS 0.0.22, so I want to know how to activate it or does it happen automatically?

**Comment by LIQUIDMIND111** [comment]
WHEN I GENERATE, I also get this issue at the beginning:

  warnings.warn(
C:\Users\Jengibre\pinokio\api\svd.pinokio.git\generative-models\venv\lib\site-packages\torch\utils\checkpoint.py:61: UserWarning: None of the inputs 
have requires_grad=True. Gradients will be None
  warnings.warn(
